python/MOCK_controller_and_boat.py

```diff
@@ -10,17 +10,29 @@
 
 # Constants
 GPS_PUBLISH_PERIOD_SECONDS = 0.1  # Keep below 1.0 for smoother boat motion
+BOAT_SPEED_KMPH = 14.4  # Boat should move at about 4m/s = 14.4 km/h
+TURNING_GAIN = 0.25  # Proportional term in the heading controller
+SPEED_CHANGE_GAIN = 0.05  # The same, but for the speed
+
 # Bring this value to about 1.0 for limited deviation. 20.0 for quite large deviations.
 HEADING_DEGREES_STANDARD_DEVIATION = 1.0
 
+# TODO: Move this to utilities
```

Comment:
Good idea to move this to utilities :)

Reply:
If moved to utilities, might be good to specify degrees in func name or var name